src/plugins/bankofgeorgia-business-ge/__tests__/converters/transactions/transaction.test.ts

```diff
@@ -228,8 +228,13 @@ describe('convertToZenMoneyTransaction', () => {
         },
         {
           id: null,
-          account: { id: 'GE00BG0000000000000000USD' },
-          sum: -40,
+          account: {
+            type: AccountType.checking,
+            instrument: 'USD',
+            company: { id: 'GE00BG0000000000000000GEL' },
+            syncIds: null
```

Comment:
Please move an account number to syncIds and leave company: null, if you don't exactly know which bank is transfer being made to. In this case you should use an inner Zenmoney bank id as company.id (you can take it from <company> tag in ZenmoneyManifest.xml).